src/test/java/org/jabref/logic/util/io/FileUtilTest.java

```diff
@@ -355,9 +356,29 @@ public void testGetLinkedDirNameDefaultFullTitle() {
                 FileUtil.createDirNameFromPattern(null, entry, fileNamePattern));
     }
 
-    private Path createTemporaryTestFile(String name) throws IOException {
-        File testFile = temporaryFolder.newFile(name);
-        Files.write(testFile.toPath(), name.getBytes(StandardCharsets.UTF_8), StandardOpenOption.APPEND);
-        return testFile.toPath();
+    @AfterEach void delete() throws IOException{
```

Comment:
here also, is this really necessary?

Reply:
And yes, those folders didn't delete themselves automatically. Every time I run this test, I'll get something like tens of temp folders.